misk/src/main/kotlin/misk/clustering/fake/ExplicitClusterResourceMapper.kt

```diff
@@ -0,0 +1,36 @@
+package misk.clustering.fake
+
+import misk.clustering.Cluster
+import misk.clustering.ClusterResourceMapper
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.atomic.AtomicReference
+
+/**
+ * An [ExplicitClusterResourceMapper] is a [ClusterResourceMapper] where the mapping
+ * is explicit managed.
+ */
+class ExplicitClusterResourceMapper : ClusterResourceMapper {
```

Comment:
does anything use this? would it make sense for the FakeCluster to use it so the test is more explicit?

Reply:
Things _will_ use it, yes.